index.js

```diff
@@ -3,15 +3,19 @@
 const minBy = require('lodash/minBy')
 const maxBy = require('lodash/maxBy')
 
-const validateProfile = require('./lib/validate-profile')
 const defaultProfile = require('./lib/default-profile')
+const createParseBitmask = require('./parse/products-bitmask')
+const createFormatProductsFilter = require('./format/products-filter')
+const validateProfile = require('./lib/validate-profile')
 const _request = require('./lib/request')
 
 const isObj = o => o !== null && 'object' === typeof o && !Array.isArray(o)
 const isNonEmptyString = str => 'string' === typeof str && str.length > 0
 
 const createClient = (profile, request = _request) => {
 	profile = Object.assign({}, defaultProfile, profile)
+	profile.parseProducts = createParseBitmask(profile)
+	profile.formatProductsFilter = createFormatProductsFilter(profile)
```

Comment:
Wouldn't this make me unable to overwrite those two functions with my own profile? Is this intended?

Reply:
you're right. will change it.